mobile/src/main/java/org/openhab/habdroid/background/BackgroundTasksManager.kt

```diff
@@ -258,7 +259,7 @@ class BackgroundTasksManager : BroadcastReceiver() {
             KNOWN_PERIODIC_KEYS.forEach { key -> scheduleWorker(context, key) }
         }
 
-        private fun managePeriodicTrigger(context: Context) {
+        fun managePeriodicTrigger(context: Context, interval: Int = 0) {
```

Comment:
Instead of `interval` there could be a parameter `forceReschedule`, but then this func must be called after https://github.com/openhab/openhab-android/pull/1927/files#diff-fddfefe814b155eac25c5f7d615ea9a8R658 has finished.

Reply:
I think that'd be a better solution. Currently, the semantics of `interval == 0` is a bit unclear. Maybe rename that method in the process to `fun schedulePeriodicTrigger(context: Context, force: Boolean)`?

Reply:
Done